scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/tasks/qe_test_requires.yml

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
You probably already thought of this and dismissed it because of reasons, but this new file looks very similar to `roles/send_to_os_health/tasks/os_health_requires.yml`. "Merging" them could save you some trouble later.

Reply:
As we discussed, importing tasks from other roles will result in role dependency.

Besides that, today when we make a change on a specific role we are sure that those changes will affect only this role. If we start importing tasks between roles we will lose this and a change on a specific role can make other roles brake.

Reply:
That makes sense, thanks.